tools/kimi_k3/benchmark_nospec_decode.py

```
@@ -0,0 +1,164 @@
+import argparse
+import hashlib
+import json
+import statistics
+import time
+import urllib.request
+from pathlib import Path
+from typing import Any
+
+
+def _get(url: str) -> bytes:
+    with urllib.request.urlopen(url, timeout=30) as response:
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)